part_model/dataloader/util.py

```diff
@@ -60,5 +60,632 @@
         [0.0667, 0.8392, 1.0000],
         [1.0, 1.0, 0.0],
         [0.2392, 0.1882, 0.9059],
+        [0.9569, 0.7373, 0.4431],
```

Comment:
We probably want a way to deterministically generate all the colors in the future. Need a context that set random seed within that context only.